core/src/services/monoiofs/backend.rs

```diff
@@ -121,8 +124,72 @@ impl Access for MonoiofsBackend {
         am.set_scheme(Scheme::Monoiofs)
             .set_root(&self.core.root().to_string_lossy())
             .set_native_capability(Capability {
+                stat: true,
+                read: true,
+                write: true,
+                delete: true,
                 ..Default::default()
             });
         am.into()
     }
+
+    async fn stat(&self, path: &str, _args: OpStat) -> Result<RpStat> {
+        let path = self.core.prepare_path(path);
+        // TODO: borrowed from FsBackend because statx support for monoio
+        // is not released yet, but stat capability is required for read
+        // and write
+        let meta = tokio::fs::metadata(&path).await.map_err(new_std_io_error)?;
```

Comment:
I didn't feel good about introducing `tokio` here. I believe we can wait for next monoio release instead.